api/setup.py

```diff
@@ -57,7 +57,11 @@ def recur_expand(target_root, dir):
                   "examples/subscribe",
                   "examples/unsubscribe",
                   ])] + list(recur_expand('share/zulip', 'integrations/')),
-    scripts=["bin/zulip-send"],
+    entry_points={
+        'console_scripts': [
+            'zulip-send=zulip.send:main',
```

Comment:
@eeshangarg `zulip-send` is also used in the shell, so you need to update the doc https://github.com/zulip/zulip/blob/master/api/README.md with `python zulip/send.py`.

http://python-packaging.readthedocs.io/en/latest/command-line-scripts.html says
> This method has the advantage that it’s very easily testable.